tests/testthat/helper-db.R

```diff
@@ -0,0 +1,88 @@
+get_db_info <- function() {
+  list(
+    dbname = "vimpact_test_db",
+    user = "postgres",
+    host = "localhost"
+  )
+}
+
+prepare_example_postgres_db <- function() {
+  info <- get_db_info()
+  vimpact_test_postgres_connection(info$dbname, info$user, info$host)
+  con <- get_test_connection()
+  add_dummy_data(con)
+  invisible(TRUE)
+}
+
+get_test_connection <- function() {
+  info <- get_db_info()
+  con <- get_postgres_connection(info$dbname, info$user, info$host)
+  withr::defer_parent(DBI::dbDisconnect(con))
+  con
+}
+
+vimpact_test_postgres_connection <- function(dbname, user, host) {
+  tryCatch(
+    get_postgres_connection(dbname, user, host),
```

Comment:
Great - we discussed doing this; this is perfect

Reply:
That wasn't actually being called from the tests and instead being called from the pkg load which wasn't ideal. I've fixed this up now